AssertionError on import

After installing via `pip`, import fails as follows:

```python
import graphenv
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Input In [1], in <cell line: 1>()
----> 1 import graphenv

File ~/.local/lib/python3.9/site-packages/graphenv/__init__.py:10, in <module>
      7     __version_tuple__ = (0, 0, "unknown version")
      9 tf1, tf, tfv = try_import_tf()
---> 10 assert tfv == 2
     11 if not tf1.executing_eagerly():
     12     tf1.enable_eager_execution()

AssertionError: 
```

Here version 2 of tensorflow seems to be required but this requirement was not handled or remarked by `pip install graphenv` and there is no proper error message describing what went wrong.

(For assertions you can give a failure message with `assert tfv == 2, failure_message`. Though, I can also see an exception being used here given that an outdated dependency can be understood as a user error rather than a developer error.)

--- a/graphenv/__init__.py
+++ b/graphenv/__init__.py
@@ -7,7 +7,9 @@ except ImportError:
     __version_tuple__ = (0, 0, "unknown version")
 
 tf1, tf, tfv = try_import_tf()
-assert tfv == 2
+
+if tf is not None:
+    assert tfv == 2, "GraphEnv only supports tensorflow 2.x"
 if not tf1.executing_eagerly():
     tf1.enable_eager_execution()
 

